modules/mobile/app/serializers/mobile/v1/user_serializer.rb

```diff
@@ -0,0 +1,142 @@
+# frozen_string_literal: true
+
+require 'fast_jsonapi'
+
+module Mobile
+  module V1
+    class UserSerializer
+      include FastJsonapi::ObjectSerializer
+
+      ADDRESS_KEYS = %i[
+        id
+        address_line1
+        address_line2
+        address_line3
+        address_pou
+        address_type
+        city
+        country_code
+        country_code_iso3
+        international_postal_code
+        province
+        state_code
+        zip_code
+        zip_code_suffix
+      ].freeze
+
+      EMAIL_KEYS = %i[
+        id
+        email_address
+      ].freeze
+
+      PHONE_KEYS = %i[
+        id
+        area_code
+        country_code
+        extension
+        phone_number
+        phone_type
+      ].freeze
+
+      SERVICE_DICTIONARY = {
+        appeals: :appeals,
+        appointments: :vaos,
+        claims: :evss,
+        directDepositBenefits: %i[evss ppiu],
+        disabilityRating: :evss,
+        lettersAndDocuments: :evss,
+        militaryServiceHistory: :emis,
+        userProfileUpdate: :vet360,
+        secureMessaging: :mhv_messaging
+      }.freeze
+
+      set_type :user
+      attributes :profile, :authorized_services, :health
+
+      attr_reader :user, :facility_names, :contact_info
+
+      # Serializes user data after first fetching user's facility and contact information
+      #
+      # @example initialization:
+      #   Mobile::V0::UserSerializer.new(@current_user, options)
+      def initialize(user, options = {})
+        @user = user
+        fetch_additional_resources
+        resource = UserStruct.new(user.id, profile, authorized_services, health)
+        super(resource, options)
+      end
+
+      private
+
+      def filter_keys(value, keys)
+        value&.to_h&.slice(*keys)
+      end
+
+      def profile
```

Comment:
Are we able to have this serializer extend the v0 serializer and just override this profile method to remove gender and fax?

Reply:
I'll give a try. Good call